model/src/w3metamd.F90

```diff
@@ -1,3 +1,20 @@
+!> @file w3metamd.F90
```

Comment:
There's no need to put the filename, and it should not be there.

Filenames can change, and doxygen will automatically adjust. So putting the file name in the comments just invites confusion in the future, when someone changes the name of the file, but forgets to update this line of documentation.

Reply:
Yes I agree.  I'll make the change when Hera is back online.  I believe I'm to blame here.  I initially wrote up our reference sheet with the filename filled in, likely adding some confusion.